package.json

```diff
@@ -216,7 +216,7 @@
     }
   },
   "devDependencies": {
-    "@types/node": "^17.0.23",
+    "@types/node": "^14.0.0",
```

Comment:
Unrelated to this change: Will dependabot try to bump the major version?

Reply:
Good call!

I found that you can control which kinds of updates you take for specific packages here: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore

Added config so that it won't suggest major updates to `@types/node`.